fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -182,3 +219,35 @@ def load_data(self, X, y, **kwargs):
         super().load_data(X, y,
                           event=pd.Series(y).apply(lambda y: _LABEL + "=" + str(y)),
                           **kwargs)
+
+
+class ErrorRateRatio(ConditionalSelectionRate):
+    r"""Implementation of Error Rate Ratio as a moment.
+
+    Measures the ratio in errors per attribute by overall error.
+    The 2-sided version of error ratio can be written as
+    ratio <= error(A=a) / total_error <= 1/ratio
+    .. math::
+    ratio <= E[abs(h(x) - y)| A = a] / E[abs(h(x) - y)] <= 1/ratio\; \forall a
+
+    This implementation of :class:`ConditionalSelectionRate` defines a single event, `all`.
+    Consequently, the `prob_event` :class:`pandas:pandas.DataFrame` will only have a single
+    entry, which will be equal to 1.
+
+    The `index` property will have twice as many entries (corresponding to the Lagrange multipliers
+    for positive and negative constraints) as there are unique values for the sensitive feature.
+
+    The :meth:`signed_weights` method will compute the costs according to Example 3 of
+    `Agarwal et al. (2018) <https://arxiv.org/abs/1803.02453>`_.
```

Comment:
"similar to Example 3" and at the end add something like "but with g=abs(h(x)-y) rather than g=h(x)".